centos2ol.sh

```diff
@@ -272,6 +314,75 @@ rm -f "${reposdir}/switch-to-oraclelinux.repo"
 # At this point, the switch is completed.
 trap - ERR
 
+# When an additional enabled CentOS repository has a match with Oracle Linux
+#  then automatically enable the OL repository to ensure the RPM is maintained
+#
+# Create an associate array where the key is the CentOS reponame and the value
+#  contains the method of getting the content (Enable a repo or install an RPM)
+#  and the details of the repo or RPM
+case "$os_version" in
+    6*)
+        declare -A repositories=(
+            [base]="REPO ol6_u10_base"
+            [base-debuginfo]="REPO https://oss.oracle.com/ol6/debuginfo/"
+            [updates]="REPO ol6_latest"
+        )
+        ;;
+    7*)
+        declare -A repositories=(
+            [base]="REPO ol7_u9_base"
```

Comment:
I match `base` to the Oracle Linux equivalent but I can't think of the advantage it brings. 
Do you have any disagreement to me ignoring `base` @Djelibeybi ?

Reply:
Yeah, I can't think of a reason to keep it. Perhaps a compromise: match base repos, but disable it. If it provides an advantage to someone else, they can enable it.